ui/app/components/app/signature-request/signature-request.container.js

```diff
@@ -1,33 +1,26 @@
 import { connect } from 'react-redux'
 import SignatureRequest from './signature-request.component'
-import { goHome } from '../../../store/actions'
 import { clearConfirmTransaction } from '../../../ducks/confirm-transaction/confirm-transaction.duck'
 import {
-  getSelectedAccount,
-  getCurrentAccountWithSendEtherInfo,
-  getSelectedAddress,
   accountsWithSendEtherInfoSelector,
-  conversionRateSelector,
 } from '../../../selectors/selectors.js'
+import { getAccountByAddress } from '../../../helpers/utils/util'
 
 function mapStateToProps (state) {
   return {
-    balance: getSelectedAccount(state).balance,
-    selectedAccount: getCurrentAccountWithSendEtherInfo(state),
```

Comment:
`selectedAccount` is required within the component

Reply:
IIRC, `accountsWithSendEtherInfoSelector` gets the same information that `selectedAccount` did, and I've replaced the latter with `fromAccount` in the component.

Reply:
Whoops, I misread this, you're right.